doc-Community_Collaboration_Guide/topics/creating_a_new_repository.adoc

```diff
@@ -1,11 +1,153 @@
 === Configuring Content Synchronization
 
 Placeholder text.
+
+[[ccg-create-repo-synch-methods]]
+==== Synchronization Methods
+
+The following synchronization methods are currently recommended:
+
+* <<ccg-create-repo-synch-batchcommit,Batch Commit>>
+
+
+[[ccg-create-repo-synch-batchcommit]]
+==== Batch Commit Synchronization Method
+
+[[ccg-create-repo-synch-batchcommit-when]]
+===== When to Use this Approach
+
+The batch commit process is a good approach to use when you need to incorporate upstream content into product documentation and the product contains a good deal of additional product-specific content.
+In other words, when the upstream content is included as a _subset_ of the product documentation.
+
+[[ccg-create-repo-synch-batchcommit-prereq]]
+===== Prerequisites
+
+The batch commit method assumes that your upstream repository is hosted on GitHub and the machine where you run the batch commit script has the following utilities installed:
+
+* `curl`
+* `zip`
+
+[[ccg-create-repo-synch-batchcommit-dir]]
+===== Directory Structure
+
+The batch commit example described here assumes that you have a book directory, `my-sample-book`, structured as follows:
+
+----
+my-sample-book/
+|__ fetch-upstream.sh
+|__ docinfo.xml
+|__ master.adoc
+|__ topics/
+|__ upstream/
+----
+
+Where `fetch-upstream.sh` is the batch commit script (for details, see <<ccg-create-repo-synch-batchcommit-script>>), and `upstream` is a subdirectory reserved for the upstream content.
+
+[[ccg-create-repo-synch-batchcommit-workflow]]
+===== Workflow
+
+Given that you already have a batch commit script, `fetch-upstream.sh`, the workflow for synchronizing content from the upstream repository is as follows:
+
+. Pull down the latest content from the upstream repository and put the downloaded files in the `upstream/` subdirectory:
++
+----
+$ ./fetch-upstream.sh
+----
+
+. Inspect the downloaded content (for example, using `git diff`) to check for new files, restructured content, and so on.
+If necessary, edit the book file, `master.adoc`, to incorporate new content.
+
+. Add and commit the downloaded content:
++
+----
+$ git add upstream
+$ git commit -m "Synched content from upstream"
+----
+
+[[ccg-create-repo-synch-batchcommit-script]]
+===== Sample Batch Commit Script
+
+The batch commit script, `fetch-upstream.sh`, is a simple `bash` script that pulls down the latest upstream content, copying the relevant upstream files into the `upstream` subdirectory.
+The script exploits a particular feature of GitHub, whereby GitHub provides a downloadable archive file containing the working tree for any branch in the repository.
+The archive file contains a snapshot of the _latest_ content in a particular branch and thus provides an ideal basis for synchronizing content downstream.
+
+To see how his works, consider the following sample `fetch-upstream.sh` script that was developed for synchronizing the contents of the _Apache Camel Component Reference_:
```

Comment:
To see how **this** works, consider the following sample `fetch-upstream.sh` script developed for synchronizing the contents of the _Apache Camel Component Reference_

Reply:
Yes, definitely better.